contracts/PrepaidCardManager.sol

```diff
@@ -550,8 +573,39 @@ contract PrepaidCardManager is Ownable, Versionable, Safe {
     return true;
   }
 
+  function createCardWithIssuer(
+    address customer,
+    address issuer,
+    address depot, // the issuers safe
+    address from,
+    address token,
+    uint256 issuingTokenAmount,
+    uint256 spendAmount,
+    string memory customizationDID
+  ) internal returns (address) {
+    // TODO: only v2 contract can call this function
```

Comment:
Should we set an `AddressSet` of trusted addresses that can call this function? And check whether it contains the PrepaidCardMarketV2's address? Or directly check if `from == prepaid card market v2 address`?